atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbTimestampStoreInvalidatorCreationTest.java

```diff
@@ -75,35 +71,15 @@ public void setUp() {
         invalidationRunner.createTableIfDoesNotExist();
     }
 
-    @Test
-    public void doesNotInvalidateMultiSeriesTable() {
-        assertThatThrownBy(() -> storeUpperLimitAndGetTimestampStoreInvalidator(Optional.of(
-                        DbTimestampCreationSettings.multipleSeries(otherTable, TimestampSeries.of("test")))))
-                .isInstanceOf(SafeIllegalStateException.class)
-                .hasMessageContaining("Invalidator must only be called by embedded DB timeLock that does not support "
-                        + "multi series timestamp store. This is unexpected, please contact support.");
-    }
-
     @Test
     public void canInvalidatorForSingleSeriesTable() {
-        TimestampStoreInvalidator timestampStoreInvalidator = storeUpperLimitAndGetTimestampStoreInvalidator(
-                Optional.of(DbTimestampCreationSettings.singleSeries(Optional.of(otherTable))));
+        TimestampStoreInvalidator timestampStoreInvalidator =
+                storeUpperLimitAndGetTimestampStoreInvalidator(Optional.of(otherTable));
         assertThat(timestampStoreInvalidator.backupAndInvalidate()).isEqualTo(TIMESTAMP_1);
 
         assertBoundNotReadableAfterBeingPoisoned(otherStore);
     }
 
-    @Test
```

Comment:
Why did we remove this test?

Reply:
Since the changes you can't physically create a config where single series is not implied. So this test becomes equivalent to `invalidatesDefaultTableForEmptyParameters()` below it